src/components/TimeseriesDataExport/stories/format-label.md

```diff
@@ -0,0 +1,29 @@
+# Format Timeseries Label
+
+<!-- STORY -->
+
+#### Usage:
+
+```typescript jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { GetTimeSeriesMetadataDTO } from '@cognite/sdk'; 
+import { TimeseriesDataExport, LabelFormatterForCSV } from '@cognite/gearbox';
+
+function ExampleComponent(props) {
+  const labelFormatter: LabelFormatterForCSV = 
+    (ts: GetTimeSeriesMetadataDTO) => ts.name || \`timeserie-${ts.id}\`; 
```

Comment:
```suggestion
    (ts: GetTimeSeriesMetadataDTO) => ts.name; 
```
we know that there is only one timeserie, and it has the name right?

Reply:
yes, but `GetTimeSeriesMetadataDTO ` interface  describes `name` field as optional. So I've placed this default string here

Reply:
ah, right